developers/weaviate/concepts/resources.md

```diff
@@ -77,9 +77,7 @@ In rare situations - typically on large machines with very high import speeds -
 
 The following tactics can help to reduce Weaviate's memory usage:
 
-- **Use vector compression**. Product quantization (PQ) is a technique that reduces the size of vectors. Vector compression impacts recall performance, so we recommend tests this setting on your dataset before using it in production.
-
-To enable PQ, update `pq` under `vectorIndexConfig` in your collection definition. For more information, see [Product Quantization](../concepts/vector-index.md#hnsw-with-product-quantization-pq).
+- **Use vector compression**. Product quantization (PQ) is a technique that reduces the size of vectors. Vector compression impacts recall performance, so we recommend testing PQ on your dataset before using it in production. <br/><br/> For more information, see [Product Quantization](../concepts/vector-index.md#hnsw-with-product-quantization-pq). <br/> To configure PQ, see [Compression](/developers/weaviate/configuration/pq-compression). 
```

Comment:
Is there a reason to use `<br/><br/>` rather than new newlines? Newlines read better & are easier to use imo.